core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -7603,6 +7603,39 @@ void checkArrayConcatAggFuncFails(SqlTester t) {
     tester.checkAgg("intersection(x)", values3, "[0, 1, 1]", 0);
   }
 
+  @Test void testModeFunc() {
+    tester.setFor(SqlStdOperatorTable.MODE, VM_EXPAND);
+    tester.checkFails(
+        "mode(^*^)", "Unknown identifier '\\*'", false);
+    strictTester.checkFails(
+        "^mode()^",
+        "Invalid number of arguments to function 'MODE'. Was expecting 1 arguments",
```

Comment:
nit: 1 argument?

Reply:
Yes. First thanks for your review. You can find related info [CALCITE-4661](https://issues.apache.org/jira/projects/CALCITE/issues/CALCITE-4661). The MODE function in calcite doesn't the same as PostgreSQL, Just a normal agg function.

Reply:
I see. Thanks for the explanation.
I mean the message should be "1 argument"? Anyway, this is a minor issue, so you may choose to accept it or not.

Reply:
Sorry. I get it.